packages/ndla-ui/src/RadioButtonGroup/RadioButtonGroup.tsx

```diff
@@ -64,20 +81,20 @@ class RadioButtonGroup extends Component {
       </section>
     );
   }
-}
 
-RadioButtonGroup.propTypes = {
-  selected: PropTypes.string,
-  options: PropTypes.arrayOf(
-    PropTypes.shape({
-      title: PropTypes.string.isRequired,
-      value: PropTypes.string.isRequired,
-      disabled: PropTypes.bool,
-    }),
-  ).isRequired,
-  label: PropTypes.string,
-  uniqeIds: PropTypes.bool,
-  onChange: PropTypes.func.isRequired,
-};
+  static propTypes = {
+    selected: PropTypes.string,
+    options: PropTypes.arrayOf(
+      PropTypes.shape({
+        title: PropTypes.string.isRequired,
+        value: PropTypes.string.isRequired,
+        disabled: PropTypes.bool,
+      }),
+    ).isRequired,
+    label: PropTypes.string,
+    uniqeIds: PropTypes.bool,
+    onChange: PropTypes.func.isRequired,
+  };
+}
```

Comment:
må du beholde denne?

Reply:
Denne fila er brukt av både javascript og typescript filer, selvom selve fila blir gjort om til typescript så er det fortsatt javascript filer som også skal bruke denne og da vil den `PropTypes` kunne gi linter feil til js filer om eventuelle props mangler (Lærte selv idag)

Reply:
Da lærte jeg også noe nytt

Reply:
da er det jo bare å gjøre alt som bruker denne om til ts filer da :-))))

Reply:
Det Bartek sa, men jeg sjekka ed nå og fant bare `.ts*` filer som brukte den. Vi mister fortsatt runtimesjekkinga, men vet ikke om vi bryr oss så veldig om det?

Whatchuthink @elenaand @N4G1

Reply:
Hva med andre pakker fra frontend-packages? Bruker de radiobuttons? isåfall er de javascript? hvis ikke kan du bare fjerne propTypen tenker jeg.

Reply:
Brukes fra js i designmanualen, men det er jo bare eksempler.

Reply:
Jeg fjernet de. Om vi har lyst best of both worlds så finnes det en babel-plugin som genererer prop-types fra typescripten våres. Kanskje noe å se på?
https://github.com/milesj/babel-plugin-typescript-to-proptypes

Reply:
Kule greier denne her. Hvis vi vil innføre runetime checks kan vi også bruke [Typescript typeguards](https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards), spesielt på api-endepunkter dersom datatypen er feil definert, eller at data fra endepunktet har endret seg.